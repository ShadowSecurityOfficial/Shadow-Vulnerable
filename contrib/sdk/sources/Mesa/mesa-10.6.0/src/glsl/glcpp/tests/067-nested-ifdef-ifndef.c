version https://git-lfs.github.com/spec/v1
oid sha256:b7f7747fae61362500e7cd6b4967cc7561e4e963e83fb9c48cfe4200846bf2d5
size 440
