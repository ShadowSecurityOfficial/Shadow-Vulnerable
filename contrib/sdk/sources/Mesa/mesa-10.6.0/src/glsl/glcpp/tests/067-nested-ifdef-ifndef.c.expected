version https://git-lfs.github.com/spec/v1
oid sha256:d2e9f5bfcfbfee19bd861cabddb5907b05c495feec14da60f3dded27ffa615f3
size 68
